blankiety_galantow/core/deck.py

```diff
@@ -33,12 +42,17 @@ def get_cards_amount(self):
 
 
 class WhiteCard:
+    id: int
+    text: str
     def __init__(self, id, deck_reader):
         self.id = id
         self.text = deck_reader["treść"]
```

Comment:
Skąd wy wytrzasnęliście tę praktykę dodawania domyślnych pól w klasie, które trzymają jedynie typ. Nadawajcie te typy normalnie xD
```suggestion
class WhiteCard:
    def __init__(self, card_id: int, deck_reader):
        self.id: int = id
        self.text: str = deck_reader["treść"]
```

Reply:
Ok, w oficjalnych docach Pythona jest przykład, gdzie wyciągają zmienną z konstruktora i definiują jej typ na poziomie klasy: https://www.python.org/dev/peps/pep-0526/#id9

Tak czy inaczej ustalanie typu w konstruktorze działa tak samo i nie powtarzamy zmiennych. Kwestia gustu pewnie